features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreaksFragment.kt

```diff
@@ -83,4 +98,43 @@ class FreaksFragment : Fragment() {
             repeat(FREAKS_COUNT) { this.add(freak) }
         }
     }
+
+    private fun loadFilters(activeFilter: String) =
+        if (activeFilter == "Skills") {
+            listOf("Android", "Kotlin", "Other Skill", "iOS")
+        } else {
+            listOf("Freaks Catalog", "Proj2", "Tutorial", "Altkeva")
+        }
+
+    private fun showFilterModal(activeFilter: String, otherView: ViewGroup?){
+        val dialog = BottomSheetDialog(requireContext())
+        val view = layoutInflater.inflate(R.layout.bottom_sheet_dialog, otherView)
+        dialog.setCancelable(true)
+        val filterTitle: TextView? = view.findViewById(R.id.filter_title)
+        filterTitle?.text = activeFilter
+        val btReset: TextView? = view.findViewById(R.id.reset)
+        val btApply: Button? = view.findViewById(R.id.apply_btn)
+        val recyclerFiltersView = view?.findViewById<RecyclerView>(R.id.recycler_filters_view)
+        val filtersList = loadFilters(activeFilter)
+        recyclerFiltersView?.layoutManager = LinearLayoutManager(view.context)
+
+        val dividerItemDecoration = DividerItemDecoration(
+            recyclerFiltersView?.getContext(),
+            (recyclerFiltersView?.layoutManager as LinearLayoutManager).getOrientation()
```

Comment:
Android studio says: Use of getter method instead of property access syntax
I really believe he is right, what do you think?